wp-content/themes/k1-theme/templates/page-employee-retention-credit.php

```diff
@@ -0,0 +1,74 @@
+<?php 
+
+/** 
+ * Page: ERC
+ */
+
+$content = new Content_Sections();
+k1_enqueue_page_style('employeeRetentionCredit');
+?>
+<section class="calculator">
+    <h3 class="color-erc-purple text-center">Estimate Your Employee Retention Credit Today!</h3>
+    <p class="text-center">We’re so glad you’re interested in learning more about the Employee Retention Credit available <br> to you through the Federal Government. Please complete this brief form so our ERC Team can <br> connect with you for your next step in the ERC process.</p>
```

Comment:
Couple of things:

- Don't use `<br/>` tags for block text (generally, and also basically always for K1). There are usually better ways to handle responsive typography.
- when using the `text-center` class, you should probably only set it at the large `lg` breakpoint (i.e. `text-lg-center`) so that it is still left-aligned on small screens.
  - This improves readability on small screens but keeps the design for desktops
  - The exception is if text (on small screens) is only 2 lines.